onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -343,7 +354,22 @@ def add_reduce_min_max(tensor_name, reduce_op_name):
                     f"Unable to guess tensor type for tensor {tensor_name!r}, "
                     f"running shape inference before quantization may resolve this issue."
                 )
-            self.model.graph.output.append(helper.make_tensor_value_info(reduce_output, onnx_type, [1]))
+
+            # Include axes in reduce_op when per_channel, always keeping axis=1
```

Comment:
Why always keep axis = 1?

Reply:
Because this is the channel-axis: to calibrate/quantize per axis, we need to reduce all dimension except channel one.